commands/util.go

```diff
@@ -80,6 +81,13 @@ func driversForNodeGroup(ctx context.Context, dockerCli command.Cli, ng *store.N
 				defer func() {
 					dis[i] = di
 				}()
+
+				buildkitAPI, err := buildkitclient.New(ctx, n.Endpoint)
```

Comment:
i think we're initializing both `buildkitAPI` and `dockerapi` here if we set `--driver=env`. So `dockerapi` initializing unnecessarily. But couldn't find a better way.

Reply:
This is definitely not allowed. Only the `driver.Client()` is allowed to call this. `dockerapi` initialization is for the features like `--load` that move things to Docker.

 Another difference is that `clientForEndpoint` is just a struct setup, while `buildkitclient.New()` actually dials out to the endpoint.